### PR TITLE
Feature/section media content

### PR DESCRIPTION
**Pull request by gergana95**
Add: html syntax to section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -77,6 +77,62 @@ <h1 class="primary-slider__title font-secondary">Beautiful Escape</h1>
...
+    <div class="section-media-content">
+        <div class="quote">
+            <p class="quote__content">Surfing is the most blissful experience you can have on this planet, a taste of heaven.</p>
+            <span class="quote__author">john mccarthy</span>
+
+            <svg class="quote__line" xmlns="http://www.w3.org/2000/svg" width="2" height="100" viewBox="0 0 2 100">
+                <path fill="#5da8ef" d="M0 100V0h2v100z"/>
+            </svg>
+
+        </div>
```
Comment (superKalo) [inline index.html]: Hm, is there any semantic HTML5 tag that represents a quote?

File: index.html [hunk 2]
```diff
@@ -77,6 +77,62 @@ <h1 class="primary-slider__title font-secondary">Beautiful Escape</h1>
...
     </div>
 
+    <div class="section-media-content">
+        <div class="quote">
+            <p class="quote__content">Surfing is the most blissful experience you can have on this planet, a taste of heaven.</p>
+            <span class="quote__author">john mccarthy</span>
+
+            <svg class="quote__line" xmlns="http://www.w3.org/2000/svg" width="2" height="100" viewBox="0 0 2 100">
+                <path fill="#5da8ef" d="M0 100V0h2v100z"/>
+            </svg>
```
Comment (superKalo) [inline index.html]: Is there any way how we can make a line in CSS, without using SVG?

File: index.html [hunk 3]
```diff
@@ -77,6 +77,62 @@ <h1 class="primary-slider__title font-secondary">Beautiful Escape</h1>
...
+
+        </div>
+
+        <div class="gallery">
+            <img class="gallery__left-image" src="images/surfer.jpg" alt="surfer image" />
+            <img class="gallery__right-image" src="images/surfer-waves.jpg" alt="surfer on waves image" />
```
Comment (superKalo) [inline index.html]: `left` and `right` are not very good naming practices in this case. Let's discuss why.

File: index.html [hunk 4]
```diff
@@ -77,6 +77,62 @@ <h1 class="primary-slider__title font-secondary">Beautiful Escape</h1>
...
+                    <filter id="a" width="200%" height="200%" x="-50%" y="-50%">
+                    <feMorphology in="SourceGraphic" operator="dilate" result="SvgjsFeMorphology1010Out"/>
+                    <feOffset dx="-2.1" dy="-2.1" in="SvgjsFeMorphology1010Out" result="SvgjsFeOffset1011Out"/>
+                    <feGaussianBlur in="SvgjsFeOffset1011Out" result="SvgjsFeGaussianBlur1012Out" stdDeviation="17.5"/>
+                    <feComposite in="SvgjsFeGaussianBlur1012Out" in2="SourceAlpha" operator="out" result="SvgjsFeComposite1013Out"/>
+                    </filter>
+                </defs>
+                <path fill-opacity=".13" d="M978 1495a34 34 0 1 1 0 68 34 34 0 0 1 0-68z" filter="url(#a)" transform="translate(-906.9 -1457.9)"/>
+                <path fill="#5da8ef" d="M71.1 37.1a34 34 0 1 1 0 68 34 34 0 0 1 0-68z"/>
+            </svg>
```
Comment (superKalo) [inline index.html]: Is this the circle? I want to see it written in pure CSS only.

File: index.html [hunk 5]
```diff
@@ -77,6 +77,62 @@ <h1 class="primary-slider__title font-secondary">Beautiful Escape</h1>
...
+                <defs>
+                    <linearGradient id="a" x1="275.5" x2="275.5" y1="1767" y2="1769" gradientUnits="userSpaceOnUse">
+                    <stop offset="0" stop-color="#fff" stop-opacity=".41"/>
+                    <stop offset=".5" stop-color="#fff" stop-opacity="0"/>
+                    <stop offset="1" stop-color="#fff" stop-opacity="0"/>
+                    </linearGradient>
+                </defs>
+                <path fill="#5da8ef" d="M0 2V0h99v2z"/>
+                <path fill="url(#a)" d="M226 1769v-2h99v2z" transform="translate(-226 -1767)"/>
+            </svg>
```
Comment (superKalo) [inline index.html]: No need for an SVG resource here.

File: index.html (hunk 1 above)
Comment (gergana95) [reply index.html]: Used q element

> The HTML < q > element indicates that the enclosed text is a short inline quotation. This element is intended for short quotations that don't require paragraph breaks; for long quotations use the < blockquote > element.

File: index.html (hunk 2 above)
Comment (gergana95) [reply index.html]: going to use after in css

File: index.html (hunk 3 above)
Comment (gergana95) [reply index.html]: renamed to col-1 and col-2

File: index.html (hunk 4 above)
Comment (gergana95) [reply index.html]: added div, will be coded with css

File: index.html (hunk 5 above)
Comment (gergana95) [reply index.html]: added div, will be coded with css

File: index.html [hunk 6]
```diff
@@ -77,6 +77,34 @@ <h1 class="primary-slider__title font-secondary">Beautiful Escape</h1>
         </div>
     </div>
 
+    <div class="section-media-content">
+
+        <div class="quote">
+            <q class="quote__content">Surfing is the most blissful experience you can have on this planet, a taste of heaven.</q>
```
Comment (superKalo) [inline index.html]: Why you choose `<q>` tag?

File: index.css [hunk 7]
```diff
@@ -211,6 +201,141 @@ body {
...
+    display: inline-block;
+    color: #333333;
+    font-size: 9px;
+    font-weight: 700;
+    text-transform: uppercase;
+    text-decoration: none;
+    letter-spacing: 5px;
+    padding-bottom: 7px;
+    position: relative;
+}
```
Comment (superKalo) [inline index.css]: Please add a cool hover effect.

File: index.css [hunk 8]
```diff
@@ -211,6 +201,141 @@ body {
...
+.gallery__play {
+    width: 68px;
+    height: 68px;
+    box-shadow: -2px -2px 35px rgba(0, 0, 0, 0.13);
+    background-color: #5da8ef;
+    border-radius: 50%;
+    position: absolute;
+    z-index: 2;
+    bottom: 40%;
+    right: 27%;
```
Comment (superKalo) [inline index.css]: This looks like a magic centering and will cause troubles during the responsive phase.

File: index.css [hunk 9]
```diff
@@ -211,6 +201,141 @@ body {
...
+    text-align: center;
+    height: 100px;
+    width: 2px;
+    margin: 0 auto;
+    background-color: #5da8ef;
+}
+
+.gallery {
+    position: relative;
+    height: 592px;
```
Comment (superKalo) [inline index.css]: Do it flexible instead.

File: index.html [hunk 10]
```diff
@@ -77,6 +77,31 @@ <h1 class="primary-slider__title font-secondary">Beautiful Escape</h1>
...
+        <article class="gallery">
+            <img class="gallery__image-col-1" src="images/surfer.jpg" alt="surfer image" />
+
+            <div class="gallery__col-2">
+                <img class="gallery__image-col-2" src="images/surfer-waves.jpg" alt="surfer on waves image" />
+                <a class="gallery__play" href="#"></a>
+            </div>
+
+            <div class="gallery__content">
+                <span class="gallery__text font-tertiary"> By better understanding the various aspects of surfing, you will improve faster and have more fun in the water. </span>
```
Comment (superKalo) [inline index.html]: Hm, `<span>` or `<p>` is better here?

File: index.css [hunk 11]
```diff
@@ -211,6 +201,168 @@ body {
...
+    height: 68px;
+    box-shadow: -2px -2px 35px rgba(0, 0, 0, 0.13);
+    background-color: #5da8ef;
+    border-radius: 50%;
+    position: absolute;
+    z-index: 2;
+    bottom: 50%;
+    right: 50%;
+    margin: 0 -34px -34px 0;
+  /*  transition: fill 4s;*/
```
Comment (superKalo) [inline index.css]: Hm, old comment?

File: index.css [hunk 12]
```diff
@@ -211,6 +201,168 @@ body {
...
+
+.gallery__col-2 {
+    position: absolute;
+    right: 0;
+    top: 45px;
+}
+
+.gallery__image-col-2 {
+    z-index: 1;
+}
```
Comment (superKalo) [inline index.css]: Z-index works only with CSS position defined (different than the default one).

File: index.css [hunk 13]
```diff
@@ -211,6 +201,168 @@ body {
...
+    border-left: 14px solid white;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    margin: -8px 0 0 -4px;
+}
+
+.gallery__play:hover::after {
+ /* maybe hover effect with triangle */
+}
```
Comment (superKalo) [inline index.css]: Maybe an abandoned comment :-)

**Comment by superKalo** [comment]
All right, looks good now! 👍 💯

**Pull request merged**